controllers/operands/kubevirt_test.go

```diff
@@ -2797,6 +2797,47 @@ Version: 1.2.3`)
 			})
 		})
 
+		Context("Virtual machine options", func() {
+			It("should set disableFreePageReporting by default", func() {
+				kv, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(kv.Spec.Configuration).To(Not(BeNil()))
+				Expect(kv.Spec.Configuration.VirtualMachineOptions).To(Not(BeNil()))
+				Expect(kv.Spec.Configuration.VirtualMachineOptions.DisableFreePageReporting).To(Not(BeNil()))
+			})
+
+			It("should modify disableFreePageReporting according to HCO CR", func() {
+
+				hco.Spec.VirtualMachineOptions = &hcov1beta1.VirtualMachineOptions{DisableFreePageReporting: true}
```

Comment:
If the default is `true` - there is no change here

Reply:
Right, actually this line can be entirely dropped since `existingResource, err := NewKubeVirt(hco)`,  as you said, already set `disableFreePageReporting` to true.

Reply:
Cool. But I think we don't have a test for DisableFreePageReporting = false

Reply:
Added